mutant/modules/artic_illumina/start.py

```diff
@@ -52,16 +52,28 @@ def run_case(self, resdir):
         if self.config_artic != "":
             confline = "-C {0}".format(self.config_artic)
 
-        cmd = "nextflow {0} -log {1} run {2} {3}/externals/gms-artic/main.nf -profile {4} --illumina --prefix {5} --directory {6} {7}".format(
-            confline,
-            nflog,
-            workline,
-            self.WD,
-            self.profiles,
-            self.prefix,
-            self.fastq,
-            resultsline,
-        )
+        if nanopore:
+            cmd = "nextflow {0} -log {1} run {2}/externals/gms-artic/main.nf -profile singularity,slurm --medaka --prefix {3} --basecalled_fastq {4} --scheme-directory {5}/externals/gms-artic/primer-schemes/midnight/nCoV-2019/V1 {6}".format(
+                confline,
+                nflog,
+                self.WD,
+                self.prefix,
+                self.fastq,
```

Comment:
Could use self.profiles instead to have the option of changing them

Reply:
ok, just making notes to myself that right know `-profile singularity,slurm` works